src/scripts/onerepStatsAlert.js

```diff
@@ -0,0 +1,83 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
@rhelmer the issue with relying a cronjob is that if / when it fails, we will have stale data and we can end up letting in more people than needed. 
1) The frequency of the runs need to be in sync with OneRep's schedule of updates (everyday)
2) We need to set up alerting for the cronjob

Reply:
> @rhelmer the issue with relying a cronjob is that if / when it fails, we will have stale data and we can end up letting in more people than needed.
> 
>     1. The frequency of the runs need to be in sync with OneRep's schedule of updates (everyday)

Does the stats API only update daily? I assumed it was closer to real-time than that.

>     2. We need to set up alerting for the cronjob

I'll add it to Sentry.

Reply:
Yes, I think the API is updated probably near real time.

Reply:
Looks like we have Sentry `captureCheckIn`, is there any other monitoring you had in mind? And yes I think it makes sense to just use this for monitoring, the pages that need to check the real-time status should just query the stats API independently.